apis/python/CMakeLists.txt

```diff
@@ -0,0 +1,332 @@
+)
+
+# Install the extension module
+install(TARGETS pytiledbsoma DESTINATION tiledbsoma)
+
+# Debug: Check if libraries exist before installing
+if(APPLE)
+    message(STATUS "Checking for: ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledbsoma.dylib")
+elseif(UNIX)
+    message(STATUS "Checking for: ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledbsoma.so")
+    message(STATUS "Checking for: ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64/libtiledbsoma.so")
+endif()
+
+# Install shared libraries - try both lib and lib64 locations
+if(APPLE)
+    install(FILES ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledbsoma.dylib
+        DESTINATION tiledbsoma
+    )
+elseif(UNIX)
+    # Install from whichever location exists (lib or lib64)
+    install(CODE "
+        if(EXISTS \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledbsoma.so\")
+            file(INSTALL \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledbsoma.so\" 
+                 DESTINATION \"\${CMAKE_INSTALL_PREFIX}/tiledbsoma\")
+            message(STATUS \"Installed libtiledbsoma.so from lib/\")
+        elseif(EXISTS \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64/libtiledbsoma.so\")
+            file(INSTALL \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64/libtiledbsoma.so\" 
+                 DESTINATION \"\${CMAKE_INSTALL_PREFIX}/tiledbsoma\")
+            message(STATUS \"Installed libtiledbsoma.so from lib64/\")
+        else()
+            message(FATAL_ERROR \"Could not find libtiledbsoma.so in ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib or ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64\")
+        endif()
+    ")
+endif()
+
+# Install TileDB shared libraries if not provided externally
+# Note: On macOS, delocate-wheel will automatically bundle dylib dependencies,
+# so we don't need to install them manually (would cause duplication)
+if(NOT DEFINED ENV{TILEDB_PATH})
+    if(APPLE)
+        # macOS: Skip installing libtiledb.dylib - delocate-wheel will handle it
+        message(STATUS "Skipping libtiledb.dylib install on macOS - delocate-wheel will bundle it automatically")
+    elseif(UNIX)
+        # Linux: Must install libtiledb.so - auditwheel needs it in place
+        install(CODE "
+            if(EXISTS \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledb.so\")
+                file(INSTALL \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib/libtiledb.so\" 
+                     DESTINATION \"\${CMAKE_INSTALL_PREFIX}/tiledbsoma\")
+                message(STATUS \"Installed libtiledb.so from lib/\")
+            elseif(EXISTS \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64/libtiledb.so\")
+                file(INSTALL \"${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64/libtiledb.so\" 
+                     DESTINATION \"\${CMAKE_INSTALL_PREFIX}/tiledbsoma\")
+                message(STATUS \"Installed libtiledb.so from lib64/\")
+            else()
+                message(FATAL_ERROR \"Could not find libtiledb.so in ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib or ${LIBTILEDBSOMA_INSTALL_PREFIX}/lib64\")
+            endif()
+        ")
+    endif()
+endif()
```

Comment:
When building wheels, I'd rather statically link libtiledb into libtiledbsoma; it would make things more predictable, and avoid relying on pip rewriting the symbol names. Or even better, statically link libtiledb, libtiledbsoma, and all of their dependencies, into pytiledbsoma.